### PR TITLE
Feature/home cards

### LINKED ISSUE #62: When home card clicked (top 15 cities) it goes to the Location page
- [ ] Add-in functionality to click a card on Home page
- [ ] When the card is clicked, display the Location (details) page for that city.

- Pass props from Home to Card (lat and long)

### LINKED ISSUE #65: Limit the number of fetch calls on Home
- Add in local storage so that when the user navigates back to Home, the data does not have to be fetched again.
- Maybe when page loads we can update state to show firstLoad to be true or something?

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- I added functionality so that when the Home page loads it stores the 15 cards data. The user can now go about the site and that data is not fetched every time the user returns Home
- I added functionality so that when the user clicks any of the cards from the Home page, the Location page loads with the details for that city
- I fixed a test for Home. I know we need to write more. It'll be easier when the data displays are agreed upon and locked in.
- Closes #65 Closes #62 Makes progress on #56 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
- The testing is mainly done navigating around the app to see that everything works properly.

# Message for reviewer:
- After we merge these last couple things in, I think it would be good for us all to focus on getting solid testing coverage.

# Checklist:
- [ ] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Home/Home.tsx [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
-		makeCards(cityData)
+		if (props.homePageData) {
+			makeCards(props.homePageData)
+		} else {
+			const getCityData = homeCities.map(async (city: {locality: string, region: string, lat: number, long: number}) => {
+				const cityLocality = city.locality
+				const cityRegion = city.region
+				const cityLat = city.lat
+				const cityLong = city.long
+				const data = await getHomeData(city.lat, city.long)
```
Comment (ErinUntermeyer) [inline src/Home/Home.tsx:33]: You can toggle this back to line 34 if you are working on styling so that the data isn't being fetched upon load.

Comment (leighlars) [reply src/Home/Home.tsx:33]: thank you!

File: src/Card/Card.tsx [hunk 2]
```diff
@@ -161,27 +166,30 @@ const Card: React.FC<CardProps> = (props) => {
...
-      <p className="unit">UVI</p>
-      {/* {aqIndex(props.aqi, props.aqiCat)}
-      <p className="unit">AQI</p> */}
-     </div>
-     {weatherIcon(props.icon)}
-    </article>
-   </Link>
-  )
+		<Link
+			to={`/results/${props.locality}`}
```
Comment (leighlars) [inline src/Card/Card.tsx:171]: clutch! I dig it

**Review by leighlars — COMMENTED** [review]
Looks great! Thank you!!!!!!

File: src/helpers/apiCalls.tsx [hunk 3]
```diff
@@ -1,8 +1,7 @@
 const airQualityKey = 'DACE2187-D810-4B4B-81E4-45AEBAF087A0'
-const weatherDataKey = "346025de560bd5d671b73a88bec2fce6"
-// second weatherDataKey 7b9afd68afe01b32e10130a572c8c564
-// 
-//'02a1de26e83e798e7595a522e8e4e9d9'
+const weatherDataKey = '02a1de26e83e798e7595a522e8e4e9d9'
+const weatherDataKey2 = '7b9afd68afe01b32e10130a572c8c564'
```
Comment (JoshSevy) [inline src/helpers/apiCalls.tsx:3]: Haha Love it!

**Comment by JoshSevy** [comment]
Great work! Looks Awesome.

**Review by JoshSevy — APPROVED** [review]

**Pull request merged**